pkg/detectors/databrickstoken/databrickstoken.go

```diff
@@ -21,52 +21,54 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	domain = regexp.MustCompile(`\b(https:\/\/[a-z0-9-]+\.cloud\.databricks\.com)\b`)
-	keyPat = regexp.MustCompile(`\b(dapi[a-z0-9]{32})\b`)
+	keyPat    = regexp.MustCompile(`\b(dapi[0-9Aa-f]{32})\b`)
+	domainPat = regexp.MustCompile(`\b([\w-\.]+\.(cloud\.databricks\.com|gcp\.databricks\.com|azuredatabricks\.net))\b`)
 )
 
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
 func (s Scanner) Keywords() []string {
-	return []string{"databricks", "dapi"}
+	return []string{"dapi"}
 }
 
-// FromData will find and optionally verify Databrickstoken secrets in a given set of bytes.
+// FromData will find and optionally verify DatabricksToken secrets in a given set of bytes.
 func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (results []detectors.Result, err error) {
-	dataStr := string(data)
 
-	matches := keyPat.FindAllStringSubmatch(dataStr, -1)
-	domainMatches := domain.FindAllStringSubmatch(dataStr, -1)
+	matches := keyPat.FindAll(data, -1)
+	domainMatches := domainPat.FindAll(data, -1)
 
 	for _, match := range matches {
-		if len(match) != 2 {
-			continue
-		}
-		resMatch := strings.TrimSpace(match[1])
 
-		for _, domainmatch := range domainMatches {
-			if len(domainmatch) != 2 {
-				continue
-			}
-			resDomainMatch := strings.TrimSpace(domainmatch[1])
+		resMatch := strings.TrimSpace(string(match))
+
+		for _, domainMatch := range domainMatches {
+
+			domain := strings.TrimSpace(string(domainMatch))
 
 			s1 := detectors.Result{
 				DetectorType: detectorspb.DetectorType_DatabricksToken,
 				Raw:          []byte(resMatch),
-				RawV2:        []byte(resMatch + resDomainMatch),
```

Comment:
Why did you remove `RawV2`?

Reply:
I wrote this detector long-time ago , not sure if RawV2 was there. I will add it. and test the code again